proto/ocpaths.proto

```diff
@@ -26,6 +26,19 @@ package openconfig.ocpaths;
 
 option go_package = "github.com/openconfig/featureprofiles/proto/ocpaths_go_proto;ocpaths";
 
+// Release is at minimum a timestamp for associated with an artifact.
+// By convention, itt is expected to uniquely identify and artifact
```

Comment:
a few typos on this line (`itt` `and artifact`)

Reply:
who knew that i get paid to type. :face-palm: